<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWidget</class>
 <widget class="QWidget" name="ClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="ip">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ip</string>
   </property>
  </widget>
  <widget class="QLabel" name="port">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>40</y>
     <width>30</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>40</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>8888</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>9</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="receive">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>71</y>
     <width>491</width>
     <height>182</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>259</y>
     <width>491</width>
     <height>182</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="send">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>447</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>424</x>
     <y>447</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connect">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
